### ISSUE TITLE
A script to create a dummy datatable for dev purposes is lacking

### ISSUE THREAD

**Issue by Bachibouzouk** [OpenEnergyPlatform/oeplatform#496]
# Problem

After a fresh install it would be nice to run a script which would create a few example table with the test user credentials.

**Comment by Bachibouzouk** [comment]
@wingechr, @gnn wouldn't you be able to do so in no time :)

**Comment by MGlauer** [comment]
Data we need for testing: 
* Some geo data for the maps (Points, Polygons)
* some time series for the graphs

**Comment by wingechr** [comment]
@MGlauer, @Bachibouzouk I would be happy to do that.

Unfortunately, in the current development branch, insertion of data via REST API seems to be broken.

The error thrown is `'IntegrityError('null value in column "_autocheck" violates not-null constraint)` I am still trying to figure out why. I set up my test environment according to the README.md

The same error also occurs when running `python manage.py test`

**Comment by MGlauer** [comment]
@wingechr This is already fixed and in review.  See #493

**Comment by Bachibouzouk** [comment]
> @wingechr This is already fixed and in review. See #493
@MGlauer I approved it yesterday, it can be merged (just add a line in the changelog before)

**Comment by Bachibouzouk** [comment]
@wingechr - #493 is merged into `develop`, you can go ahead :)

**Comment by 4lm** [comment]
Yes, I need this (dev data)! Thanks @wingechr for working on this!

**Comment by 4lm** [comment]
If the test data also could contain geodata, this would be nice!

**Comment by Bachibouzouk** [comment]
@wingechr - we are now (at least) two people awaiting this feature :), when do you think you can do it?

**Comment by Bachibouzouk** [comment]
@wingechr, @MGlauer - isn't this something you can do fast? To quickly make some tests it would be very useful (and also very good that we all use the same reference test datasets)

**Comment by jh-RLI** [comment]
@wingechr @Ludee @chrwm @steull 
We should create dummy data that can be added to the fresh OEDB install. I would also like to integrate this into the docker container. 

Lets collect some usefull data together. 
- [ ] timeseries 
- [ ] scalars
- [ ] text
- [ ] geo data

I have already created [some test data](https://github.com/OpenEnergyPlatform/oedatamodel/tree/develop/examples) for the oedatamodel tables (data from the open_MODEX project). We could reuse these and create some simple but jet-realistic tables and add them to all available schemas after the database setup is done.  It would be useful to add appropriate metadata as well.

**Comment by chrwm** [comment]
> I have already created [some test data](https://github.com/OpenEnergyPlatform/oedatamodel/tree/develop/examples)

I could add some additional rows with additional technological parameters (to not only have installed capacity). The same goes for timeseries.csv. Let me know if you think it's not necessary.

**Comment by jh-RLI** [comment]
Very nice :) I can create a script to create the tables via the API on the local development version of the OEP and then upload the data. I will also help with creating the test data. Perhaps @steull can also help create the test data and create the metadata once the test datasets are available? This data/tables could also be used to test the review process (I think). 

I think it is enough to create single tables (tables without forigen keys/relations), the example for the "oedatamodel" already covers tables with relations.

**Comment by jh-RLI** [comment]
To complete this for our use case, we should try to add data that matches the schemas/data categories on the OEP (which could result in different data sets for each schema). This is also helpful when we use the data to test the review process, and provides good examples of what data to expect (for developers).

**Comment by Ludee** [comment]
These are suitable for testing:

- [model_draft.oep_metadata_table_example_v151](https://openenergy-platform.org/dataedit/view/model_draft/oep_metadata_table_example_v151)
- [boundaries.bkg_vg250_1_sta](https://openenergy-platform.org/dataedit/view/boundaries/bkg_vg250_1_sta)
